src/client/fdsn.py

```diff
@@ -108,7 +111,6 @@ def _request(url, post=False, user=None, passwd=None,
     if url_values:
         url += '?' + url_values
     logger.debug('Accessing URL %s' % url)
-
     url_args = {
         'timeout': g_timeout
```

Comment:
To have `kwargs` overwrite the timeout we can use
`url_args.update(**kwargs)`

Reply:
Right now fdsn.g_timeout = 60. overwrites config and default values.